utils/packaging/cmake/BundlePlugins.cmake

```diff
@@ -0,0 +1,55 @@
+set(XDG_ROOT ${CMAKE_BINARY_DIR}/xdg_root)
+set(LVIM_XDG_DATA_HOME ${XDG_ROOT}/share)
+set(LVIM_XDG_CONFIG_HOME ${XDG_ROOT}/.config)
+set(LVIM_XDG_CACHE_HOME ${XDG_ROOT}/.cache)
```

Comment:
I don't have the full picture but I feel like this would require moving the plugins again since `xdg_root` is hardcoded in the path. 

This should be easy to verify though, so please show the intended folder structure.